File: 450-DSA-SHEET/ARRAYS/C-Plus-Plus/3-Kth_Smallest_or_Largest_Element_in_an_Array.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int n; 
    cout << "Enter the number of elements: ";
    cin>>n;
    int arr[n];
    cout << "Enter the elements of the array: ";
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }
    int k;
    cout << "Enter the Kth value to find the Smallest/Largest Value in an array: " << endl;
    cin>>k;
    
    return 0;
}

File: 450-DSA-SHEET/ARRAYS/Python/3-Kth_Smallest_or_Largest_Element_in_an_Array.py
def kthsmallest(arr,n,k):
    arr.sort()
    return arr[k-1]

if __name__ == '__main__':
    arr = [12, 3, 5, 7, 19]
    n = len(arr)
    k = 2
    print("The kth Smallest/Largest Element in the array is: ", kthsmallest(arr, n, k))

